pyupgrade.py

```diff
@@ -1537,6 +1535,37 @@ def generic_visit(self, node):  # type: (ast.AST) -> None
         super(FindPy3Plus, self).generic_visit(node)
 
 
+def reversed_walk(node):  # type: (ast.AST) -> Generator[ast.AST, None, None]
+    todo = [node]
+    while todo:
+        node = todo.pop()
+        todo.extend(ast.iter_child_nodes(node))
+        yield node
+
+
+def targets_referenced_after_node(root_node, node, targets):
+    # type: (ast.AST, ast.AST, Union[ast.expr, ast.Tuple]) -> bool
+    """
+    Search inside the given root node whether the targets have been referenced
```

Comment:
docstrings should not start with whitespace

Reply:
Addressed in 81dc036f6a52e32a3dd1e992334c24fc09cea12b.